llvm/test/Verifier/noalias-addrspace.ll

```diff
@@ -0,0 +1,60 @@
+; RUN: not llvm-as < %s -o /dev/null 2>&1 | FileCheck %s
+
+; CHECK: It should have at least one range!
```

Comment:
You may want to include a test case with a valid range.

Reply:
Those are in test/Assembler